### ISSUE TITLE
Handing in Big Customer / Out of Curiosity / Chemical Pt. 4 causes double rep loss

### ISSUE THREAD

**Issue by refringe** [sp-tarkov/server#674]
However, it appears to have a side effect of causing double the rep loss when handing in any of these 3 quests now.

_Originally posted by @Aburiu in https://dev.sp-tarkov.com/SPT-AKI/Issues/issues/278#issuecomment-21276_

---

[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/281) by _**chomp**_

**Comment by refringe** [comment]
Fixed with: https://dev.sp-tarkov.com/SPT-AKI/Server/commit/47a84e508b7a6418002ab1e0637811e94f535615

This may break some quests with wait times not appearing correctly (show instantly instead of after x hours)

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/281#issuecomment-21536) by _**chomp**_

**Comment by refringe** [comment]
From testing so far, it looks good, Gunsmith Part 6 has not instantly shown up after completing Part 5.

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/281#issuecomment-21573) by _**Aburiu**_

**Comment by refringe** [comment]
Just had the quest Chemical Part 4 apply the 0.25 rep loss twice on a build from Feb 29 (https://dev.sp-tarkov.com/SPT-AKI/Server/commit/c64e5f72399e1d2cf8e30be00c95b3a737db6070).
For some reason, I don't have logs in `BepInEx/` and `Logs/`, so I'm only attaching `server-*.log`

This was fixed with https://dev.sp-tarkov.com/SPT-AKI/Server/commit/47a84e508b7a6418002ab1e0637811e94f535615 but I think https://dev.sp-tarkov.com/SPT-AKI/Server/commit/64810f72166bb9e665485323f44b3ed6b5b0e2eb reintroduced the bug.


---
### Attachments
- [server-2024-03-10.log](https://dev.sp-tarkov.com/attachments/e498bced-044f-4ade-aa7f-822319ff1470) (229 KiB)


---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/281#issuecomment-24543) by _**belette**_

**Comment by refringe** [comment]
Can't reproduce on Mar 10 build. Are there any special steps?

---
### Attachments
- [2024-03-10[19-58] (0).png](https://dev.sp-tarkov.com/attachments/21512120-4f5b-4ba7-930e-fd1af9eeff4f) (3165 KiB)


---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/281#issuecomment-24548) by _**5o2**_

**Comment by refringe** [comment]
Tested on latest commit as well, unable to reproduce this
![image](https://dev.sp-tarkov.com/attachments/18ddc7d8-93b8-497a-9508-3dc9ddce864b)

---
### Attachments
- [image.png](https://dev.sp-tarkov.com/attachments/18ddc7d8-93b8-497a-9508-3dc9ddce864b) (209 KiB)


---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/281#issuecomment-24550) by _**DrakiaXYZ**_

**Comment by refringe** [comment]
Thanks for looking into it.

My steps were:
- go for the Therapist variant ([Out of Curiosity](https://escapefromtarkov.fandom.com/wiki/Out_of_Curiosity)) by planting a marker in Customs
- survive the raid
- complete quest at Therapist
- observe -0.50 rep at Prapor and Skier instead of -0.25

I'd be happy to try and repro it on latest bleeding edge build if I can get pointers on how to quickly get a new account to this quest state. Do we have debug commands I can use or is it all manual fiddling with save files?

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/281#issuecomment-24551) by _**belette**_

**Comment by refringe** [comment]
You can create a new profile with the "SPT Developer" type, select USec, and all quests will be available to accept.

Tested by handing in to Therapist, still only reduced rep one time for the other traders
![image](https://dev.sp-tarkov.com/attachments/ca696b36-5143-4412-ae7b-ce4b381b1078)


---
### Attachments
- [image.png](https://dev.sp-tarkov.com/attachments/ca696b36-5143-4412-ae7b-ce4b381b1078) (126 KiB)


---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/281#issuecomment-24552) by _**DrakiaXYZ**_

**Comment by refringe** [comment]
Thank you, I will try to repro on both builds and report back.

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/281#issuecomment-24553) by _**belette**_

**Comment by refringe** [comment]
Welp, I managed to repro once on the Feb29 version, could not repro on Mar10, then could not repro on Feb29 anymore... I tried doing it in raid too, etc. but no dice. Maybe I had some cache/bad state from earlier bleeding edge versions, I'm not sure. In any case, I learned a lot about the project so hopefully my future reports will come with consistent repro steps.

This seems fixed and can be closed, thank you for your time.

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/281#issuecomment-24563) by _**belette**_
